clients/cobol-lsp-vscode-extension/src/services/snippetcompletion/cobolSnippets.json

```diff
@@ -96,55 +203,197 @@
+  },
   "COMPUTE EQUAL": {
     "prefix": "COMPUTE EQUAL",
     "body": ["COMPUTE ${1:id} EQUAL ${2:arithExp}"],
     "description": "COMPUTE id EQUAL arithexp"
   },
+  "CONFIGURATION": {
+    "prefix": "CONFIGURATION",
+    "body": ["ENVIRONMENT DIVISION.", "CONFIGURATION SECTION."],
+    "description": "The CONFIGURATION SECTION describes the z/OS environment on which the program is compiled and executed. It is an optional section for programs, functions, prototypes, and classes. https://www.ibm.com/docs/en/cobol-zos/6.4?topic=division-configuration-section"
+  },
+  "CONTAINS": {
+    "prefix": "CONTAINS",
+    "body": [
+      "FD ${2:inputFile}",
+      "\tRECORD CONTAINS ${1:numLiteral} CHARACTERS",
+      "\tBLOCK CONTAINS 0 RECORDS",
+      "\tDATA RECORD IS ${3:inputFileRec}."
+    ],
+    "description": "A fixed-block file. Note that BLOCK CONTAINS 0 leaves the blocking-optimization up to z/OS - and is the preferred way to specify most QSAM file FDs. https://www.ibm.com/docs/en/cobol-zos/6.4?topic=entries-block-contains-clause"
+  },
   "CONTENT": {
     "prefix": "CONTENT",
     "body": ["CONTENT ${1:id}"],
     "description": "CONTENT id"
   },
+  "CONTENT-OF": {
+    "prefix": "CONTENT-OF",
+    "body": [
+      "COMPUTE ${1:resultNum} = CALC2(CONTENT-OF(${2:arg-1}),${3:arg-2},${4:arg-3})",
+      "END-COMPUTE"
+    ],
+    "description": "The CONTENT-OF intrinsic function produces a temporary copy of the input argument. https://www.ibm.com/docs/en/cobol-zos/6.4?topic=functions-content"
+  },
+  "CONTROL": {
+    "prefix": "CONTROL",
+    "body": ["*CONTROL ${1|MAP,NOMAP|)"],
+    "description": "*CONTROL is a Compiler option - that allows the user to limit (or not limit) the MAP output of a program compile.  https://publibfp.dhe.ibm.com/epubs/pdf/igy6pg40.pdf "
+  },
   "CONVERTING": {
     "prefix": "CONVERTING",
-    "body": ["CONVERTING ${1:item}"],
-    "description": "CONVERTING item"
+    "body": [
+      "INSPECT DATA-4 CONVERTING",
+      "\"${1:abc}\" TO \"${2:123}\"",
+      "AFTER INITIAL \"|\" BEFORE INITIAL \"/\"",
+      "END-INSPECT",
+      "$0"
+    ],
+    "description": "CONVERTING is a parameter of the INSPECT keyword - used to replace character values. https://www.ibm.com/docs/en/cobol-zos/6.4?topic=inspect-examples-statement"
+  },
+  "COPY": {
+    "prefix": "COPY",
+    "body": ["COPY ${1:copybookName}"],
+    "description": "COPY is used to insert source code - stored in a PDS or SCM into COBOL lines. Note that the JCL or Source Management System (Endevor, etc.) must be available and specified correctly. https://publibfp.dhe.ibm.com/epubs/pdf/igy6pg40.pdf"
```

Comment:
"COPY is used to insert source code stored in a PDS or SCM into a COBOL program when it is compiled. Note that the JCL or Source Management System (Endevor, etc.) must be available and specified correctly during compilation. https://publibfp.dhe.ibm.com/epubs/pdf/igy6pg40.pdf"

(I assume this is what is meant)